src/helpers/author-archive-helper.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * A helper object for author archives.
+ *
+ * @package Yoast\YoastSEO\Helpers
+ */
+
+namespace Yoast\WP\Free\Helpers;
+
+/**
+ * Class Date_Helper
+ */
+class Author_Archive_Helper {
+
+	/**
+	 * Gets the array of post types that are shown on an author's archive.
+	 *
+	 * @return array The post types that are shown on an author archive.
+	 */
+	public function get_author_archive_post_types() {
+		/**
+		 * Filters the array of post types that are shown on an author's archive.
+		 *
+		 * @since 12.7
```

Comment:
I think this should be removed, we don't use `@since` tags consistently, and also RC 12.7 is made today (26-11) and it's not sure whether this PR will make it through the process before.